strings/tips.txt

```diff
@@ -258,3 +258,4 @@ Just because someone attacks you doesn't mean they're trying to kill you, people
 Questions? Consult the Wiki: https://wiki.yogstation.net/wiki/Main_Page
 Questions? Consult the Mentors using the 'MHelp' system.
 Starving makes you move slower. Be sure to keep fed!
+As the Chaplain You can use your bible on the altar of god to choose a veriety of religions with special rites.
```

Comment:
As the Chaplain, You can use your bible on the altar of god to choose a variety of religions with special rites.